### DISCUSSION TITLE
Continued PreTraining Notebook of llama3.1,mistral-nemo

### DISCUSSION THREAD

**Discussion by rohithbojja** [NVIDIA-NeMo/NeMo#10620] [Q&A]
How can i perform CPT of LLama3.1,Mistral-Nemo etc using Nvidia-Nemo.

Help please

**Comment by elliottnv** [comment]
@yaoyu-33 Can you please provide some guidance? Thank you!

**Comment by palash04** [comment]
For CPT, I am facing this issue with both hf_resume as well as default_resume (added config, model path) - but getting this error - 
"required distributed checkpoint"
